app/src/components/Map/LayerBlendingOptionsTooltip.jsx

```diff
@@ -46,6 +47,23 @@ class LayerOptionsTooltip extends Component {
       opacityRangeValue: this.opacityRangeConfig.value,
       hueRangeValue: this.hueRangeConfig.value
     };
+
+    this.closeTooltip = this.closeTooltip.bind(this);
+  }
+
+  shouldComponentUpdate(nextProps) {
+    if (nextProps.visible !== this.props.visible) {
+      if (nextProps.visible) window.addEventListener('click', this.closeTooltip);
+      else window.removeEventListener('click', this.closeTooltip);
```

Comment:
I actually don't think you need to add/remove that event listener at each component update. This could be done at component mount/unmount.
Your implementation has a lot of potential for memory leaks (event listeners never cleanup)